raddb/mods-config/sql/main/mssql/queries.conf

```diff
@@ -323,10 +323,10 @@ accounting {
 				SET \
 					AcctStopTime = ${....event_timestamp}, \
 					AcctSessionTime = '%{Acct-Session-Time}', \
-					AcctInputOctets = convert(bigint, '%{%{Acct-Input-Gigawords}:-0}' * POWER(2.0, 32)) | '%{%{Acct-Input-Octets}:-0}', \
-					AcctOutputOctets = convert(bigint, '%{%{Acct-Output-Gigawords}:-0}' * POWER(2.0, 32)) | '%{%{Acct-Output-Octets}:-0}', \
+					AcctInputOctets = convert(bigint, '%{&Acct-Input-Gigawords || 0}' * POWER(2.0, 32)) | '%{&Acct-Input-Octets || 0}', \
```

Comment:
Maybe just do all this in FR?

```
AcctInputOctets = %{(((uint64) &Acct-Input-Gigawords) << 32) | (uint64) &Acct-Input-Octets}
```

That's a _lot_ simpler.  It means that we don't need the `|| 0` everywhere.  The code now goes "No Acct-Input-Gigawords?  OK, the default value is zero"